README.md

```diff
@@ -453,7 +453,12 @@ These settings will configure the proxying behavior of the following services:
     * [ecs.service](packages/ecs-agent/ecs.service)
 
 * `settings.network.https-proxy`: The HTTPS proxy server to be used by services listed above.
-* `settings.network.no-proxy`: A list of hosts that are excluded from proxying.
+* `settings.network.no-proxy`: A list of hosts that are excluded from proxying.  
+   Example: 
+   ```
+   settings.network.https-proxy = "1.2.3.4"
+   settings.network.no-proxy = ["localhost","127.0.0.1"]
```

Comment:
```suggestion
   [settings.network]
   https-proxy = "1.2.3.4:8080"
   no-proxy = ["localhost", "127.0.0.1"]
```
Would you mind a slight reformat, like this?  This gets closer to the format used elsewhere for examples, since TOML parsers tend to prefer related settings in a single table.  It also gives an example port, since that's fairly common for proxies.